presto-cli/src/main/java/com/facebook/presto/cli/KeyReader.java

```diff
@@ -32,8 +32,7 @@ public static int readKey()
             return -1;
         }
 
-        try {
-            InputStream in = new FileInputStream(FileDescriptor.in);
+        try (InputStream in = new FileInputStream(FileDescriptor.in)) {
```

Comment:
This is a false positive. The input stream is created on top of an existing descriptor, thus we don't close the stream because we don't want to close the descriptor.